packages/endo/test/main.test.js

```diff
@@ -1 +1,4 @@
 // no tests at this time
+import test from "ava";
+
+test.todo("No tests at this time");
```

Comment:
Why do we need this?

Reply:
Every package.json has a `scripts: test: ava`, which complains if there are no tests. So, we need this stub until we design this package properly.